msisdn-gateway/e2e-test/.gitignore

```diff
@@ -0,0 +1,6 @@
+*~
+.DS_Store
+*.egg-info/
+*.pyc
+__pycache__/
+*.json
```

Comment:
This is fine here, but not sure if we want to promote these up into the global .gitignore (except that sneaky *.json one).